wordSearchSolver.py

```diff
@@ -49,65 +49,64 @@ def get_input():  # get input two letters
+            for mod_coordinate in coordinates_to_check:
+                valid_coordinate_output = valid_coord(add_coordinates(coordinate_to_check, mod_coordinate))
                 # if a valid coordinate
-                if validCoordinateOutput[0]:
+                if valid_coordinate_output[0]:
                     # check if there is a match. If there is one next to first letter, append and break loop
-                    if word_grid[validCoordinateOutput[1]] == user_input[1]:
-                        combinationsDict.setdefault(coordinateToCheck, []).append(
-                            {"coordinate": validCoordinateOutput[1], "modCoordinate": modCoordinate})
-                logging.debug(
-                    "NumPossibilities: {}\nsecond_letter(): {}".format(len(combinationsDict.keys()), combinationsDict)) # combinationsDict.keys() does not work here
-                return combinationsDict
-
-        def find_answer(combinationsDict):
-
-            times_iter = (len(user_input) - 2)  # -2 becuase already did first and can't find next letter of last letter
-            for potentialStartCoordinate in combinationsDict.items():
-                # iterate over the list of posibilities that deviate from start point
-                for potentialNextCoordinateDict in enumerate(potentialStartCoordinate[1]):
-                    currentIterateCount = 0
-                    startIteratationCoordination = potentialNextCoordinateDict[1]["coordinate"]
-                    modCoordinate = potentialNextCoordinateDict[1]["modCoordinate"]
-                    while currentIterateCount < times_iter:
-                        # move up in the direction give
-                        startIteratationCoordination = addCoordinates(startIteratationCoordination, modCoordinate)
-                        if validCoordinate(startIteratationCoordination)[0] == False:
-                            break
-                        # now check if the letter is the right letter
-                        if word_grid[startIteratationCoordination] == user_input[currentIterateCount + 2]:
-                            # if it's the right number, keep going. But if it came to an end, then stop everything and return to user.
-                            return potentialStartCoordinate[0]
-                        currentIterateCount += 1
-            return "Nothing was found."
-        return find_answer(second_letter(first_letter()))
+                    if word_grid[valid_coordinate_output[1]] == user_input[1]:
+                        combinations_dict.setdefault(coordinate_to_check, []).append(
+                            {"coordinate": valid_coordinate_output[1], "mod_coordinate": mod_coordinate})
+        logging.debug(
+            "NumPossibilities: {}\nsecond_letter(): {}".format(len(combinations_dict.keys()), combinations_dict)) # combinations_dict.keys() does not work here
```

Comment:
(len(combinations_dict.keys()) does not actually work. Doesn't affect overall file performance, so not sure what to do.